src/util/Util.js

```diff
@@ -218,7 +218,7 @@ class Util extends null {
    */
   static escapeBold(text) {
     let i = 0;
-    return text.replace(/\*\*(\*)?/g, (_, match) => {
+    return text.replaceAll(/\*\*(\*)?/g, (_, match) => {
```

Comment:
Unnecessary, please revert to `replace` as RegExp with global flag will go thru all instances.
```suggestion
    return text.replace(/\*\*(\*)?/g, (_, match) => {
```